### PR TITLE
#2 - Fix template import

### PR DESCRIPTION
**Pull request by piavgh**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -72,6 +72,7 @@ module.exports = {
         '/unifiedinbox/app',
         '/unifiedinbox/api',
         '/calendar/app',
+        '/calendar/api',
```
Comment (tuanlc) [inline webpack.config.js:75]: Why? It should belong to another commit

File: src/linagora.esn.calendar/app/run.js [hunk 2]
```diff
@@ -2,9 +2,15 @@
   'use strict';
 
   angular.module('esn.calendar')
-    .run(runBlock);
+    .run(runBlock)
```
[the remark this answers is not in the document]
Comment (piavgh) [reply src/linagora.esn.calendar/app/run.js]: I've fixed this.

File: webpack.config.js (hunk 1 above)
Comment (piavgh) [reply webpack.config.js:75]: I've fixed this also

**Comment by tuanlc** [comment]
The second commit lacks issue number preference
